### DISCUSSION TITLE
Github Releases(remove latest checkbox as the default)

### DISCUSSION THREAD

**Discussion by josealdaco** [community/community#118725] [Other Feature Feedback, Questions, & Ideas]
### Github Releases

Product Feedback

### Body

Hi, we use Github Releases as a way to orchestrate our CI-CD process. If a release is marked as latest it will ship the tagged source code associated with the release to production. If a release is marked as a pre-release it will ship to our preview environment. Our workflow consist of having a pre-release ship out first and then promoted to latest once it has been tested and approved in the preview environment. Though rare we have had cases where someone would accidentally forget to manually check the pre-release box and simply click the Publish release button, by default Github Releases has the latest checkbox checked so in these scenarios a production deployment would initiate instead of going to our preview environment first.

Would it be possible to remove the defaults altogether or have the pre-release checkbox checked instead of latest by default?

**Comment by bjiusc** [comment]
Any updates here? we follow a similar release procedure and it would be nice to confdigure pre-release as the default checked box, or require the user to check a box before publishing. the current default is prone to accidents
